integration/test/chart/basic/basic_test.go

```diff
@@ -0,0 +1,145 @@
+// +build k8srequired
+
+package basic
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/giantswarm/e2e-harness/pkg/release"
+	"github.com/giantswarm/e2etemplates/pkg/chartvalues"
+
+	"github.com/giantswarm/chart-operator/integration/key"
+)
+
+// TestChartLifecycle tests a Helm release can be created, updated and deleted
+// uaing a chart CR processed by chart-operator.
+//
+// - Create chart CR using apiextensions-chart-e2e-chart.
+// - Ensure test app specfied in the chart CR is deployed.
+//
+// - Update chart CR using apiextensions-chart-e2e-chart.
+// - Ensure test app is redeployed using updated chart tarball.
+//
+// - Delete apiextensions-chart-e2e-chart.
+// - Ensure test app is deleted.
+//
```

Comment:
I added godoc for the test steps like we do in `e2etests`. I really like that approach.